doc/source/user/quickstart.rst

```diff
@@ -713,27 +713,29 @@ Several arrays can be stacked together along different axes::
 
 The function `column_stack`
 stacks 1D arrays as columns into a 2D array. It is equivalent to
-`vstack` only for 1D arrays::
+`hstack` for 2D arrays::
 
     >>> from numpy import newaxis
-    >>> np.column_stack((a,b))   # With 2D arrays
-    array([[ 8.,  8.,  1.,  8.],
-           [ 0.,  0.,  0.,  4.]])
-    >>> a = np.array([4.,2.])
+    >>> a = np.array([4.,3.])
     >>> b = np.array([2.,8.])
-    >>> a[:,newaxis]  # This allows to have a 2D columns vector
-    array([[ 4.],
-           [ 2.]])
-    >>> np.column_stack((a[:,newaxis],b[:,newaxis]))
+    >>> np.hstack((a,b))          # stacks as-is
```

Comment:
The idea was not to delete `newaxis` stuff, because it is the only place in this tutorial with examples how to use `newaxis` and I try to make some logic.